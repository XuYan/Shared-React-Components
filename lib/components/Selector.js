import React from 'react';

import styles from './SelectorStyles';

const Selector = props => {
    return (
        <div style={styles.div}>

        </div>
    );
};

export default Selector;