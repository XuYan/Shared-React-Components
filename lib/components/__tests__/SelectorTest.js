import React from 'react';
import Selector from '../Selector';

import renderer from 'react-test-renderer';

describe('Selector', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Selector />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

});