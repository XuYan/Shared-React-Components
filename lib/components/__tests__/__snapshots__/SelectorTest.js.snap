// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Selector renders correctly 1`] = `
<div
  style={
    Object {
      "border": "1px solid black",
      "height": "100px",
      "width": "100px",
    }
  }
/>
`;
